### DISCUSSION TITLE
Landkreis Kassel ICS second step

### DISCUSSION THREAD

**Discussion by birlo** [mampfes/hacs_waste_collection_schedule#2819] [Q&A] [answered]
I followed the instruction and this says: Go to https://webapp.abfall-kreis-kassel.de/abfallkalender, choose your Town and rightclick on "Kalender importieren" and save the link.

Problem is, that the town has an additional request, which Bins are taken from you, so you dont get the real link and so an error in HomeAssistant. What do i wrong?

**Comment by 5ila5** [comment, marked as answer]
Updated documentation in #2823

You need to click the download button once before copying the link as clicking the button changes the link of the button

**Reply by birlo** [reply]
Thats the solution!!! Thank you very much!!
